v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -64,6 +66,39 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
 
   @TemplateParameter.Text(
       order = 4,
+      regexes = {"(^jdbc:[a-zA-Z0-9/:@.]+$)"},
+      groupName = "Source",
+      description =
+          "Connection URL to connect to the source database host. Port number and connection properties must be supplied separately.",
+      helpText = "The JDBC connection URL string. For example, `jdbc:mysql://some-host`.")
+  String getSourceHost();
```

Comment:
Why do we need jdbc:mysql. Can we not just make do with the host and construct the rest ?

Reply:
I think this will be more relevant with pipeline controller changes.
Currently the options don't take in whether the DB is jdbc type or not, and I am not sure how we want to lay out the options to make it compatible with different sources.

I will set up some time with you to discuss this and once we have a way out we can cleanup the options better.